pkg/hostparams/hostparams.go

```diff
@@ -150,8 +154,13 @@ func New(config *viper.Viper, dbClaim *v1.DatabaseClaim) (*HostParams, error) {
 	}
 
 	if hostParams.EngineVersion == "" {
-		hostParams.isDefaultVersion = true
-		hostParams.EngineVersion = defaultEngineVersion
+		if dbClaim.Status.ActiveDB.DBVersion != "" {
+			log.Info("Using Status.ActiveDB.DBVersion as DefaultVersion")
```

Comment:
Not sure we want to log this, but Debug would be better. This is going to be logged a lot
```suggestion
			log.V(-1)Info("Using Status.ActiveDB.DBVersion as DefaultVersion")
```